tests/vxlan/test_vnet_bgp_route_precedence.py

```diff
@@ -61,11 +61,15 @@ def fixture_route_type(request):
     name="monitor_type",
     scope="module",
     params=SUPPORTED_MONITOR_TYPES)
-def fixture_monitor_type(request):
+def fixture_monitor_type(duthosts, request, rand_one_dut_hostname, tbinfo):
     '''
         This fixture forces the script to perform one monitor_type at a time.
         So this script doesn't support multiple monitor types at the same time.
     '''
+    asic_type = duthosts[rand_one_dut_hostname].facts["asic_type"]
+    topo = tbinfo["topo"]["type"]
+    if request.param in ["custom"] and asic_type in ["cisco-8000"] and topo == "t1":
```

Comment:
Please also include Mellanox platform. We saw the same issue on Mellanox as well. Thanks

Reply:
Added.